src/Compilers/Server/VBCSCompiler/NamedPipeClientConnection.cs

```diff
@@ -148,9 +149,19 @@ private NamedPipeServerStream ConstructPipe(string pipeName)
                 PipeTransmissionMode.Byte,
                 PipeOptions.Asynchronous | PipeOptions.WriteThrough,
                 PipeBufferSize, // Default input buffer
-                PipeBufferSize // Default output buffer
-            );
-            pipeStream.SetAccessControl(security);
+                PipeBufferSize, // Default output buffer
+                security,
+                HandleInheritability.None);
+#else
```

Comment:
add a link to this bug and a brief comment
https://github.com/dotnet/corefx/issues/24040

Reply:
👍, didn't see that was created already when I updated, so I just wanted to get code out before adding the comment.